test/examples/double_integrator/tests.jl

```diff
@@ -43,7 +43,11 @@ function lcvx()::Nothing
         @test true
 
         if !tests_failed
-            plot_trajectory(sol_lcvx, sol_mp, param_choice)
+            try
+                plot_trajectory(sol_lcvx, sol_mp, param_choice)  
+            catch
+                println("PyPlot Issues")
+            end
```

Comment:
I'm really not a fan of just swallowing the error with an all-encompassing message. We should at the very least print the underlying exception.